Translate readme pt br (#2)
luistak/idp-guideline: README.md

Hi there👋!

This is a guide to think and prepare an individual development plan, based on a few experiences I had in the past years

> Good preparation increases the `probability` of success and decreases the `risk` of things going wrong along the way

## 🚶 Steps

This guideline is totally based on _you_ and is composed by four continuous steps

### 💭 Understanding _your_ current moment

In this step you should start thinking about your current moment, like answering a few questions:

> - What are the levels(jr, pl, sr) in my context? (for your manager, tribe and company)

Reviewer: ```suggestion
- What are the levels(junior, middle, senior) in my context? (for your manager, tribe and company)
```

  > You can read this [awesome article](https://www.swyx.io/writing/career-ladders/) about engineering career ladders on [swyx.io](https://www.swyx.io/)

> - Where am i?

Reviewer: ```suggestion
- Where am I?
```

  > What is my actual level? Junior, Senior, Principal? It depends on your company or the current job position you are looking for

- What is my actual impact sphere? (product, team, tribe, chapter, company, market ...)

  > Understand where you can make the real difference, become noted not only by shining but helping others to do their best it's an awesome way to stand out and develop your peers

- What are my weakness and strengths?
  > Seek out for feedbacks from your peers and managers, also don't limit yourself to a unique perspective of your performance reach out for other managers, coaches and what makes sense at your current moment

### 🚩 Defining goals

Based on your current moment, you should start thinking about your next goals

> Relax this is not an easy thing to do, take a walk, watch some movies, be relaxed to understand what really warms your heart and remember: there is no perfect path, the best path is the one that suits bests for _you_ on _your_ current moment

> - Understand future goal, or most close goal to achieve (Ex: become a Sr. Engineer)

Reviewer: ```suggestion
- Understand your future goal, or your closest goal to achieve (Ex: become a Sr. Engineer)
```

I'm not sure about this phrase though: "Understand your future goal" and "closest goal to achieve" both are talking about something that will happen in the future, so it seems pure repetition of an idea.

>   > A common way to think about this is planning 2, 3, and 5 years in the future where you want to be?

Reviewer: ```suggestion
  > A common way to think about this is planning 2, 3, and 5 years in the future where you want to be
```

## ✋ Examples

> - [Notion template](https://www.notion.so/PDI-IDP-Your-Name-c541f5a6bf574d5abf3ed6c60f8e1820)

Reviewer: ```suggestion
- [Notion's template](https://www.notion.so/PDI-IDP-Your-Name-c541f5a6bf574d5abf3ed6c60f8e1820)
```
